app/models/price_group.rb

```diff
@@ -20,9 +20,17 @@ class PriceGroup < ActiveRecord::Base
   scope :for_facility, -> (facility) { where(facility_id: [nil, facility.id]) }
   scope :globals, -> { where(facility_id: nil) }
 
-  scope :base, -> { globals.where(name: Settings.price_group.name.base) }
-  scope :external, -> { globals.where(name: Settings.price_group.name.external) }
-  scope :cancer_center, -> { globals.where(name: Settings.price_group.name.cancer_center) }
+  def self.base
```

Comment:
Handing as part of a merge is fine. However, I'd like to have that branch ready to go (minus a rebase) in NU & DC before we merge this. I don't want problems from this in one of the forks on this to hold up releasing other features. So that means we should create a branch in the different schools, merge in this branch, and then see where we're at. Once we have that ready, we can do a merge from open, cherry-pick from our fork-specific branch, and issue that as a PR. Does that make sense?